packages/client/hmi-client/src/workflow/ops/model-from-code/tera-model-from-code.vue

```diff
@@ -183,14 +199,8 @@ onUnmounted(() => {
 });
 
 function buildJupyterContext() {
-	const contextName =
-		selectedModelFramework.value === ModelFramework.Decapodes ? 'decapodes_creation' : null;
-	const languageName =
-		selectedProgrammingLanguage.value === ProgrammingLanguage.Julia ? 'julia-1.9' : null;
-	if (contextName === null || languageName === null) {
-		logger.warn("Can't work with current language. Do nothing.");
-		return null;
-	}
+	const contextName = 'decapodes_creation';
+	const languageName = 'julia-1.9';
```

Comment:
statically settings these for now - We will only run the kernel when the decapodes options is selected anyways

Reply:
I agree with you, but why not leaving that nice test there?

Reply:
Youre right - I've added the tests for contextName and languageName but removed the warning and the early null return.  We still want the context to be setup when the component is mounted and returning null wouldn't let that happen